### DISCUSSION TITLE
uupool steals coins / SCAM POOL

### DISCUSSION THREAD

**Discussion by anton85798** [Chia-Network/chia-blockchain#4520] [General]
the reward that should be, according to the official page of the pool  2.12 CHIA/PB chia
I was paid for 12 TB 0.0083 chia
but should be 2,12/1000 tb=0,00212 chia/1 tb
0,00212*12=0,02544 chia

POOL steals OVER 60% COINS

**Comment by trwbox** [comment]
Well pools aren't officially supported yet, and it's not a pool from this dev, so they can't do much about it

**Comment by Dariux1991** [comment]
like 7€ per days?

**Comment by anton85798** [comment]
> Well pools aren't officially supported yet, and it's not a pool from this dev, so they can't do much about it
I just created a topic so that people would know that this pool is stealing a lot.

**Comment by climba03003** [comment]
I am not supporting any pool before the official pool protocol release.
but I think your should understand the number "2.12 Chia / PiB" is expected income but not guarantee income. It can be the pool is unlucky in that period of time and it lower your income. This situation is common for any cryptocurrency mining pool.

The concept is similar to Chia GUI "Estimate Time to Win"

**Comment by unitedchia** [comment]
The current pools are actually pretty scammy and require your private key and your mnemonic seed 24 word phrase.

The official pool protocol will be released on 05/17/2021 as stated by [Bram Cohen on Twitter](https://twitter.com/bramcohen/status/1390407484179763202).
There are already a variety of nice and fair concepts for pool farming, which do not require your private key nor your mnemonic seed 24 word phrase.

One of those pools is us. If you are interested, feel free to subscribe to our mailinglist on [chiaunited.com](https://chiaunited.com) 
The first 300 users get free pool farming for 6 months.

**Reply by Magby123** [reply]
> The current pools are actually pretty scammy and require your private key and your mnemonic seed 24 word phrase.
> 
> The official pool protocol will be released on 05/17/2021 as stated by [Bram Cohen on Twitter](https://twitter.com/bramcohen/status/1390407484179763202).
> There are already a variety of nice and fair concepts for pool farming, which do not require your private key nor your mnemonic seed 24 word phrase.
> 
> One of those pools is us. If you are interested, feel free to subscribe to our mailinglist on [chiaunited.com](https://chiaunited.com)
> The first 300 users get free pool farming for 6 months.

 YES and that will be SCAM UPDATE . All  OLD plots will be UNABLE to use new pool protocol.  FAR away to be fair ...

**Reply by SackofFluor** [reply]
ahm dude.... that was said from the very beginning....
EVERYONE with more then a single braincell said :" Be careful with these pools, it is likely that they scam you for an unreasonable high fee, wait for the official pool protocols"

But no... it was greed as always... now you guys plotted in the scampools  and you all feel cheated... well.... I am farming on 50TBips right now and I didn't get a single chia... you at least got something... so stop whining, you couldn't wait, so you paid an "express fee" to the pool :P :P 

Now people like you are trapped in the "sunken cost dilemma" throwing more investment into a bad investment.
Just stop plotting for them, reorganize your whole setup for the things you have learned and start over in the new pools...

**Reply by Magby123** [reply]
Dont know why you whining with your  useless 50Tb. Anyone with couple of brains, know-  needs at least to have 100TB+ to be able to get something solo ( 50% network ago) .  So you play with random and get  your "random".
  And now you can keep sit - cos even official pools no needs your useless 50Tb ;)  Who is more foolish and greedy?) Use additional info  and you can see ( even in wallet) that "... expect time to win with current difficulty could be x3 x4 times more..." 
   Dont be toxic - you are far away to be smart.

**Reply by room101-dev** [reply]
Why lie, nobody requires private-key to mine on hpool, and its in english. You people are crazy, sitting on your plots and earning zero chia, while everybody all along in asia was getting rich.

Insane.

**Reply by room101-dev** [reply]
> Dont know why you whining with your useless 50Tb. Anyone with couple of brains, know- needs at least to have 100TB+ to be able to get something solo ( 50% network ago) . So you play with random and get your "random".
> And now you can keep sit - cos even official pools no needs your useless 50Tb ;) Who is more foolish and greedy?) Use additional info and you can see ( even in wallet) that "... expect time to win with current difficulty could be x3 x4 times more..."
> Dont be toxic - you are far away to be smart.

BINGO I have 200 plots, and earn $20/day on Hpool, trouble off course is chia-space EB goes up 8% daily, or doubles every 9 days, so I have to double my plots to break even :( Well for now I plot 12+/day, so  its essentially the same as an ETH miner running same cost.

**Comment by JessicaX83** [comment]
You forgot to take into account the amount of chia the pool is bringing in. You can't figure it at 100% from the chia calculator because the pool is not getting 100% of the rewards.  I have been using Hpool for some time and the rewards are very close to when I figure what I should be getting. I get about 42% of what the chia calculator says I should get if I were to be solo, That's because Hpool is getting about 42% of the rewards.

If you received .0083 chia for 12 TB in a 24 hour period, taking the chia value around that time at $1200, thats nearly $1 per TB which is quite good and better than I would expect from a smaller pool.

**Reply by oytp** [reply]
> You forgot to take into account the amount of chia the pool is bringing in. You can't figure it at 100% from the chia calculator because the pool is not getting 100% of the rewards. I have been using Hpool for some time and the rewards are very close to when I figure what I should be getting. I get about 42% of what the chia calculator says I should get if I were to be solo, That's because Hpool is getting about 42% of the rewards.
> 
> If you received .0083 chia for 12 TB in a 24 hour period, taking the chia value around that time at $1200, thats nearly $1 per TB which is quite good and better than I would expect from a smaller pool.

Thanks for your reply here, dude.

But let's compare: I get 0.009 for 9TB, and you get 0.0083 for 12TB? The pool isn't honest, trust me and other users. Of course, it's better than nothing, but still.

**Reply by JessicaX83** [reply]
> > You forgot to take into account the amount of chia the pool is bringing in. You can't figure it at 100% from the chia calculator because the pool is not getting 100% of the rewards. I have been using Hpool for some time and the rewards are very close to when I figure what I should be getting. I get about 42% of what the chia calculator says I should get if I were to be solo, That's because Hpool is getting about 42% of the rewards.
> > If you received .0083 chia for 12 TB in a 24 hour period, taking the chia value around that time at $1200, thats nearly $1 per TB which is quite good and better than I would expect from a smaller pool.
> 
> Thanks for your reply here, dude.
> 
> But let's compare: I get 0.009 for 9TB, and you get 0.0083 for 12TB? The pool isn't honest, trust me and other users. Of course, it's better than nothing, but still.

Not that it matters but I am not a dude :)    Anyhow... there's a lot that needs to be taken into account.  I am gonna guess that you are new to crypto mining. It's never super consistent from day to day but over weeks and months, it is fairly consistent. UUpool is a smaller pool so there will be bigger swings, you will see some days with bigger rewards than other days. Over the course of months+, it will level off and be about the same. You just can't figure it like that in the short term. As of now, pool rewards seem to be floating around $1 per TB per day. I have been mining for a little over a year now and those figures are quite good and right about what I would have expected. 

If you take the figures from the chia calculator, input your plot space, and multiply the pool size of the network, which you may have to guess, and that will give you a rough estimate on what to expect.  I use about 42% for HPool, so whatever Chia calculator says... I am receiving about 42% of that. The numbers do match up, you just have to take everything into account. I don't think you are getting ripped off.

**Reply by oytp** [reply]
> > > You forgot to take into account the amount of chia the pool is bringing in. You can't figure it at 100% from the chia calculator because the pool is not getting 100% of the rewards. I have been using Hpool for some time and the rewards are very close to when I figure what I should be getting. I get about 42% of what the chia calculator says I should get if I were to be solo, That's because Hpool is getting about 42% of the rewards.
> > > If you received .0083 chia for 12 TB in a 24 hour period, taking the chia value around that time at $1200, thats nearly $1 per TB which is quite good and better than I would expect from a smaller pool.
> > 
> > 
> > Thanks for your reply here, dude.
> > But let's compare: I get 0.009 for 9TB, and you get 0.0083 for 12TB? The pool isn't honest, trust me and other users. Of course, it's better than nothing, but still.
> 
> Not that it matters but I am not a dude :) Anyhow... there's a lot that needs to be taken into account. I am gonna guess that you are new to crypto mining. It's never super consistent from day to day but over weeks and months, it is fairly consistent. UUpool is a smaller pool so there will be bigger swings, you will see some days with bigger rewards than other days. Over the course of months+, it will level off and be about the same. You just can't figure it like that in the short term. As of now, pool rewards seem to be floating around $1 per TB per day. I have been mining for a little over a year now and those figures are quite good and right about what I would have expected.
> 
> If you take the figures from the chia calculator, input your plot space, and multiply the pool size of the network, which you may have to guess, and that will give you a rough estimate on what to expect. I use about 42% for HPool, so whatever Chia calculator says... I am receiving about 42% of that. The numbers do match up, you just have to take everything into account. I don't think you are getting ripped off.

Dude = boy or girl, but excuse me anyway if I made you disappointed :)

Yep, I also mine other cryptos for 2 months already (ETH), and I don't expect a very high income yet, because from day to day it can raise and drop very often and high. As about chia, I'm very sad that the network grows dramatically fast, so I'm suck at 0.01/day - this was when I had 47 plots, and the same now (~100 plots). Yep, everything we could do is just make new plots and wait. And pray for less network grows :D

P.S. sorry for my bad English, I'm from Russia

**Reply by room101-dev** [reply]
> > You forgot to take into account the amount of chia the pool is bringing in. You can't figure it at 100% from the chia calculator because the pool is not getting 100% of the rewards. I have been using Hpool for some time and the rewards are very close to when I figure what I should be getting. I get about 42% of what the chia calculator says I should get if I were to be solo, That's because Hpool is getting about 42% of the rewards.
> > If you received .0083 chia for 12 TB in a 24 hour period, taking the chia value around that time at $1200, thats nearly $1 per TB which is quite good and better than I would expect from a smaller pool.
> 
> Thanks for your reply here, dude.
> 
> But let's compare: I get 0.009 for 9TB, and you get 0.0083 for 12TB? The pool isn't honest, trust me and other users. Of course, it's better than nothing, but still.

Or put another way on hpool, for my 200 plots I get one XCH every two months, ...
